autoload/lsp/utils/text_edit.vim

```diff
@@ -202,6 +202,9 @@ function! s:generate_sub_cmd_replace(text_edit) abort
     else
         let l:sub_cmd .= "\"=l:merged_text_edit['merged']['newText']\<CR>p"
     endif
+    if l:new_text =~# '^\n$'
+        let l:sub_cmd .= 'dd'
```

Comment:
If we use `dd` would this change the register/clipboard contents?

Reply:
I think it will. This might be better?

```suggestion
        let l:sub_cmd .= '"_dd'
```